_data/projects/ClippyService.yml

```diff
@@ -0,0 +1,11 @@
+name: Clippy Service
+desc: Rust Linting (with Clippy) as a Webservice
+site: http://clippy.bashy.io
+tags:
+- rust
+- compiler
+- linter
+- web
+upforgrabs:
+  name: help-wanted
+  link: https://github.com/ligthyear/clippy-service/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```

Comment:
You can simplify this to `https://github.com/ligthyear/clippy-service/labels/help%20wanted`